lib/queue.rb

```diff
@@ -1,28 +1,27 @@
 class Queue
```

Comment:
This all works, but isn't done with a circular buffer as assigned.  So many of the methods end up with O(n) time complexity instead of O(1).